Formula/atomicparsley.rb

```diff
@@ -35,4 +35,12 @@ def install
                           "--disable-universal"
     system "make", "install"
   end
+
+  test do
+    cp test_fixtures("test.m4a"), testpath/"file.m4a"
+    system "#{bin}/atomicparsley", testpath/"file.m4a", "--artist", "Homebrew"
+    output = shell_output("find . -name file-temp*")
+    var = "file-temp"
```

Comment:
Can we find a test that doesn't rely on internal details, such as looking for temporary files?